sp800-63b/sec4_aal.md

```diff
@@ -38,11 +38,13 @@ However, [Table 4-2](#63bSec4-Table2) shows the expanded set of AALs that are al
 
 | M-04-04 Level of Assurance | Authenticator Assurance Level
 |:------------------:|:-----------------------------:
-| 1 | 1, 2 or 3 
+| 1; without making personal data available | 1, 2 or 3 
+| 1; making personal data available | 2 or 3
 | 2 | 2 or 3
 | 3 | 2 or 3 
 | 4 | 3 
 
+At IAL1, it is possible that self-asserted personal data is collected and made available by the digital service.  Any personal data, self-asserted or validated, requires MFA; therefore agencies SHALL select a minimum of AAL2 when self-asserted personal data, collected at IAL1, is made available online.
```

Comment:
-63B does not already use the abbreviation MFA anywhere, so this should be spelled out.